docs/configuration.md

```diff
@@ -865,6 +865,13 @@ Apart from these, the following properties are also available, and may be useful
     configuration and setup documentation</a> for more information.
   </td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.service.name</code></td>
```

Comment:
For yarn, docs/running-on-yarn.md is more suitable for documentation?

Reply:
You're right. Will move it there.